main.go

```diff
@@ -80,7 +80,7 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.StringVar(&cfg.ImageProxy, "image-proxy", "", "Http proxy to pull image")
 	flagSet.StringVar(&cfg.User, "user", "", "user[:password] Registry user and password")
 	flagSet.BoolVar(&cfg.PrepareImage, "pull-images", false, "If this flag set, we will just pull container images")
-	flagSet.StringSliceVar(&rePullImages, "repull-images", []string{}, "Images d2p-migrator will actually pull")
+	flagSet.BoolVar(&cfg.ImageManifestOnly, "image-manifest-only", true, "Pull image manifest only or not")
```

Comment:
How about set the default value to `false`, so that users do not surprise why they can not use the images. WDYT?

Reply:
agree with you